libraries/AP_HAL_SITL/SITL_cmdline.cpp

```diff
@@ -243,6 +243,10 @@ void SITL_State::_parse_command_line(int argc, char * const argv[])
             break;
         }
     }
+    if (sitl_model == nullptr) {
```

Comment:
It's a static in the bss.  I think we try to avoid initialising things unnecessarily because of the increase in code-size.  Given this is SITL that doesn't mean much, but for consistency I didn't go hunting...